lib/provider.js

```diff
@@ -100,7 +100,7 @@ class StorageProvider {
     this.active = false;
     this.cursorFactory = null;
     this.schema = null;
-    this.systemLogger = systemLogger;
+    this.systemLogger = systemLogger || (() => {});
```

Comment:
```suggestion
    this.systemLogger = systemLogger || common.emptiness;
```
?

Reply:
What would be the point of changing this?

Reply:
IMO, it will look a bit better though I'm fine either way.